server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');
 const { MongoClient } = require('mongodb');

```

Comment:
Consider adding a comment explaining the purpose of setting HOSTNAME to '0.0.0.0'